### PR TITLE
feat(page-header): allow crumbs to be hidden

### PR DESCRIPTION
**Pull request by christieb7007**
showCrumbs prop added to show/hide breadcrumbs component. Defaults to true

Also updated the spaces -> spc url

### CODE REVIEW

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://60f0986cb93e901278bf7e2c--condescending-leavitt-66e607.netlify.app

File: packages/page-header/PageHeader.js [hunk 1]
```diff
@@ -89,7 +90,7 @@ const PageHeader = ({
   return (
     <>
       <div className="d-flex align-items-start flex-shrink-0">
-        {React.isValidElement(crumbs) ? (
+        {showCrumbs && React.isValidElement(crumbs) ? (
```
Comment (stevenviar) [inline packages/page-header/PageHeader.js]: Wouldn't this only apply when `crumbs` is passed as a React node?

Comment (christieb7007) [reply packages/page-header/PageHeader.js]: updated

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://60f198f555430228535d9c8e--condescending-leavitt-66e607.netlify.app

**Review by stevenviar — APPROVED** [review]

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://60f19ad387d39d2ab0ceaf9f--condescending-leavitt-66e607.netlify.app

**Pull request merged**